virtualcluster/pkg/syncer/conversion/helper.go

```diff
@@ -163,7 +192,28 @@ func BuildMetadata(cluster, vcns, vcname, targetNamespace string, obj client.Obj
 	return target.(client.Object), nil
 }
 
-func BuildSuperMasterNamespace(cluster, vcName, vcNamespace, vcUID string, obj client.Object) (client.Object, error) {
+func (c *objectConversion) CleanOpaqueKeys(vc *v1alpha1.VirtualCluster, keyMap map[string]string) {
+	var exceptionsList []string
+	if vc != nil {
+		exceptions := sets.NewString()
+		exceptions.Insert(vc.Spec.OpaqueMetaPrefixes...)
+		exceptions.Insert(constants.DefaultOpaqueMetaPrefix, constants.DefaultTransparentMetaPrefix)
+		exceptionsList = exceptions.UnsortedList()
+	}
+
```

Comment:
Should we check the length of the list and return directly if it is 0 (although in practice it is not 0)?

Reply:
we should also check the opaquedomain here, cannot directly return even if exceptionlist is 0